index.js

```diff
@@ -68,7 +68,7 @@ function uglifyify(file, opts) {
     if (debug && matched) {
       opts.sourceMap.content = convert.fromJSON(
         new Buffer(matched[1], 'base64').toString()
-      ).sourceMap
+      ).sourcemap
```

Comment:
is not camel-cased: https://github.com/thlorenz/convert-source-map/blob/master/index.js#L52